client/src/components/Workflow/TrsImport.vue

```diff
@@ -72,6 +76,9 @@ export default {
         hasErrorMessage() {
             return this.errorMessage != null;
         },
+        isRegistered() {
```

Comment:
I think it might be better to have this be `isAnonymous`, since not all users need to be registered to be able to import workflows (e.g. with auto-registration or other fancy/weird setups). I think in practice that's not going to be a difference, but it more closely reflects what you're checking here.